etna/home/models.py

```diff
@@ -40,6 +40,9 @@ class HomePage(AlertMixin, TeaserImageMixin, MetadataPageMixin, BasePage):
     settings_panels = BasePage.settings_panels + AlertMixin.settings_panels
     promote_panels = MetadataPageMixin.promote_panels + TeaserImageMixin.promote_panels
 
+    # DataLayerMixin overrides
+    gtm_content_group = "Homepage"
+
```

Comment:
```suggestion
    # DataLayerMixin overrides
    gtm_content_group = "Stories from the collection"

```

Reply:
Hi @TNA-Allan. Where did this value come from? Take a look at the [homepage](https://develop-sr3snxi-rasrzs7pi6sd4.uk-1.platformsh.site/) and look for the following script tag in the head:
```
<script id="gtmDatalayer" type="application/json">{"contentGroup1": "Homepage", "customDimension1": "offsite", "customDimension2": "", "customDimension3": "home page", "customDimension4": "", "customDimension5": "", "customDimension6": "", "customDimension7": "", "customDimension8": "", "customDimension9": "", "customDimension10": "", "customDimension11": "", "customDimension12": "", "customDimension13": "", "customDimension14": "", "customDimension15": "", "customDimension16": "", "customDimension17": ""}</script>
```

We need to preserve the values that display currently (they were [set here](https://github.com/nationalarchives/ds-wagtail/blob/develop/etna/core/models/basepage.py#LL19-L23C2) previously)

Reply:
The requirement says **Stories from the collection** for all story pages (including the homepage)

Reply:
I think there's some misunderstanding there. I think @hoake meant that the 'index page' for the articles, not the 'home page'.

Reply:
As revised in the requirements, contentGroup1= "stories" for all story pages i.e. Article index page, Article. No change for contentGroup1 for Home page.